### DISCUSSION TITLE
Welcome to test Discussions!

### DISCUSSION THREAD

**Discussion by turt2live** [turt2live/test#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tulir** [comment]
When will this be bridged to Matrix?

**Reply by turt2live** [reply]
probably soon

**Reply by mvgorcum** [reply]
Absolutely proprietary, but at least can be bridged to foss

**Comment by deepbluev7** [comment]
This repo is not very helpful.

**Reply by turt2live** [reply]
good

**Reply by deepbluev7** [reply]
Also, how do you handle spam?

**Reply by turt2live** [reply]
like that
